handler.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+const uuid = require('uuid');
+const AWS = require('aws-sdk'); 
+AWS.config.setPromisesDependency(require('bluebird'));
+const dynamoDb = new AWS.DynamoDB.DocumentClient();
+
+/* POST /guests */
+module.exports.submitGuest = async (event) => {
+	const requestBody = JSON.parse(event.body);
+  const firstName = requestBody.firstName;
+  const lastName = requestBody.lastName;
+  const responded = requestBody.responded;
+
+	if (typeof firstName !== 'string' || typeof lastName !== 'string' || typeof responded !== 'boolean') {
+    return {
+      statusCode: 400,
+      headers: {
+        'Access-Control-Allow-Origin': '*',
+        'Access-Control-Allow-Methods': 'OPTIONS, POST, GET',
+      },
+      body: JSON.stringify({
+        message: `Validation failed. Invalid input type(s).`,
+      })
+    }
+  }
+
+	const guest = guestItem(firstName, lastName, responded);
+
+  await putGuest(guest);  
+  return {
+    statusCode: 200,
+    headers: {
+      'Access-Control-Allow-Origin': '*',
+      'Access-Control-Allow-Methods': 'OPTIONS, POST, GET',
+    },
+    body: JSON.stringify({
+      message: `Added ${firstName} ${lastName}`,
```

Comment:
Typically, on a 200, these rest endpoints should always return the body of the noun of the resource. So in this case, rather than the API responding with, "yes this was added" as if it was a logger to stdout, it should return the object that was created. Specifically:
```
{
    "id": "<>",
    "firstName": "<>",
    "lastName": "<>",
    "responded": false,
}
```
That way, clients of this API would receive the actual instance that was created along with the "id" (identifier) that was created. For example, a website might show an "edit" page for the item that was just created.